changelog/932.feature.rst

```diff
@@ -0,0 +1 @@
+Added a function to solve for the analytical solution of the dispersion relation in low frequency regime.
```

Comment:
```suggestion
Add the function
:func:`~plasmapy.dispersion.two_fluid_dispersion.two_fluid_dispersion_solution` to
`plasmapy.dispersion`, which gives an analytical solution to the dispersion relation as
derived by P. M. Bellan 2012 (DOI: `10.1029/2012JA017856
<https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2012JA017856>`_).
```

The change log acts as an official log for us, so I like entries to be as specific/explicit as possible without losing readability.